### PR TITLE
[86bxv66aj][carousel] made all carousel container focusable

### PR DESCRIPTION
**Pull request by ilyabrower**
## Motivation and Context
We've added `keyboardFocusable` enhancer to all Carousel container and added `tabIndex=-1` to Buttons inside Carousel.

## How has this been tested?
I've tested it manually

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [X] Nice improve.

## Checklist:

- [X] My code follows the code style of this project.
- [X] I have updated the documentation accordingly or it's not required.
- [X] Unit tests are not broken.
- [X] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

File: semcore/carousel/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [3.29.0] - 2024-03-14
+
+### Added
+
+- `keyboardFocusEnhance` to Carousel component.
+- `tabIndex = -1` to Buttons in Carousel.
```
Comment (msereniti) [inline semcore/carousel/CHANGELOG.md:10]: Let's just say that you made the whole carousel focusable and prev/next button non focusable

Maybe also we need to mention changed arrows navigation?

File: website/docs/components/carousel/carousel-code.md [hunk 2]
```diff
@@ -33,6 +33,16 @@ tabs: Design('carousel'), A11y('carousel-a11y'), API('carousel-api'), Example('c
 
 :::
 
+## Carousel with focusable Indicators only
```
Comment (msereniti) [inline website/docs/components/carousel/carousel-code.md:36]: Does this carousel really contains focusable indicators?

**Review by msereniti — APPROVED** [review]
Please fix changelogs 🙏

File: semcore/carousel/CHANGELOG.md (hunk 1 above)
Comment (ilyabrower) [reply semcore/carousel/CHANGELOG.md:10]: fixed

File: website/docs/components/carousel/carousel-code.md (hunk 2 above)
Comment (ilyabrower) [reply website/docs/components/carousel/carousel-code.md:36]: no, fixed

**Pull request merged**